tutorials/io/binary_serialization_api.rst

```diff
@@ -383,13 +393,70 @@ For each Name and Sub-Name
 Every name string is padded to 4 bytes.
 
 16: :ref:`RID<class_rid>` (unsupported)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-17: :ref:`Object<class_object>` (unsupported)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+17: :ref:`Object<class_object>`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An Object could be serialized in three different ways: as a null value, with
+``full_objects = false``, or with ``full_objects = true``.
+
+A null value
+""""""""""""
+
++----------+-------+------------+-------------------------------------------------+
+| Offset   | Len   | Type       | Description                                     |
++==========+=======+============+=================================================+
+| 4        | 4     | Integer    | Zero (32-bit signed integer)                    |
++----------+-------+------------+-------------------------------------------------+
+
+``full_objects`` disabled
+"""""""""""""""""""""""""
+
++----------+-------+------------+-------------------------------------------------+
+| Offset   | Len   | Type       | Description                                     |
++==========+=======+============+=================================================+
+| 4        | 8     | Integer    | The Object instance ID (64-bit signed integer)  |
++----------+-------+------------+-------------------------------------------------+
+
+``full_objects`` enabled
+""""""""""""""""""""""""
+
++----------+-------+----------------+----------------------------------------------------------+
+| Offset   | Len   | Type           | Description                                              |
++==========+=======+================+==========================================================+
+| 4        | 4     | Integer        | Class name (String length)                               |
++----------+-------+----------------+----------------------------------------------------------+
+| 8        | X     | Bytes          | Class name (UTF-8 encoded string)                        |
++----------+-------+----------------+----------------------------------------------------------+
+| X+12     | 4     | Integer        | The number of properties that are serialized             |
```

Comment:
Offset should be X+8.

Reply:
Shouldn't it include the sum of the offsets before it?

Reply:
Offset is a sum of the preceding lengths (here possibly padded to 4 bytes). Or equivalently it's (previous offset) + (previous length).

Reply:
Oh. Of course. Not sure what I was thinking! Thanks, done.

Reply:
Looks fine now.